### DISCUSSION TITLE
how can I move items vertically center

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1519] [Q&A] [answered]
When I change the height of ui.header, items in it doesn't centered  as shown in the picture ,I just  add 'h-10'
```py
 with ui.header(elevated=True).style('background-color: #3874c8').classes('items-center h-10 justify-between'):
        ui.label('HEADER')
        ui.button(on_click=lambda: right_drawer.toggle(), icon='menu').props('flat color=white')
```
![微信截图_20230830003646](https://github.com/zauberzeug/nicegui/assets/25096045/41b03d8f-7399-4c11-aa59-64fa28e0faf8)

**Comment by falkoschindler** [comment, marked as answer]
I think there is just not enough vertical room.
Adding `py-0` to the header classes to remove the vertical padding should solve your problem.
